flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/stream/StreamPhysicalLookupJoin.scala

```diff
@@ -100,7 +100,8 @@ class StreamPhysicalLookupJoin(
     new StreamExecLookupJoin(
       tableConfig,
       JoinTypeUtil.getFlinkJoinType(joinType),
-      remainingCondition.orNull,
+      finalPreFilterCondition.orNull,
+      finalRemainingCondition.orNull,
```

Comment:
Why are `finalPreFilterCondition` and `finalRemainingCondition` both `orNull`, but in `CommonExexcLookupJoin`,  only `preFilterCondition` with  json include `NON_NULL`, but `remainingJoinCondition` without.